app/models/child.rb

```diff
@@ -165,6 +165,154 @@ def initialize *args
                     }
                  }
               }"
+
+    view :by_followup_date,
+    :map => "function(doc) {
+					if (doc['couchrest-type'] == 'Child') {
+						if (doc.hasOwnProperty('gbv_follow_up_subform_section') && doc['gbv_follow_up_subform_section'] && doc['gbv_follow_up_subform_section'].length) {
+							doc['gbv_follow_up_subform_section'].forEach(function(followUp) {
+								if (followUp['followup_date']) {
+									emit([doc['owned_by_agency'], followUp['followup_date']], null);
+								}
+							});	
+						}
+					}
+				}"
+
+    view :by_days_between_case_opening_and_closure,
+    :map => "function(doc) {
+        if (doc['couchrest-type'] == 'Child' && doc.hasOwnProperty('date_closure') && doc['date_closure'] != null) {
+          var dateOpen = new Date(doc['registration_date']);
+          var dateClosure = new Date(doc['date_closure']);
+          var oneDay = 1000 * 60 * 60 * 24;
+          var dayDifference = +((dateClosure - dateOpen) / oneDay).toFixed(0)
+          var dayRange = #{Kpi::CASE_CLOSURE_TYPE[:days_ranges]}
+            .filter(function (ranges) {
+              return dayDifference >= ranges[0] && dayDifference <= ranges[1]
+            });
+          emit([doc['owned_by_agency'], dayRange[0] ? dayRange[0][0] + '-' + dayRange[0][1] : 'other'], null)
+          emit([doc['owned_by_agency'], '_count'], null)
```

Comment:
Not sure why you would want to emit a row in the view with the string '_count'.  I tested this.  For 1 case record, 2 rows appeared... 1 for the date range and the other with _count.   Seems odd.